AirportTicketBookingExercise/Logic/Service/UserService.cs

```diff
@@ -0,0 +1,69 @@
+﻿using ATB.Data.Repository;
+using ATB.Data.Models;
+using System.Xml.Linq;
+
+namespace ATB.Logic.Service
+{
+    public class UserService : IUserService
+    {
+        private readonly IUserRepository _userRepository;
+        public UserService(IUserRepository userRepository)
+        { 
+        
+            _userRepository = userRepository;
+        }
+
+        public User? Authenticate(string username, string password, UserType usertype)
+        {
+            var user = GetUserByName(username);
+            if (user != null && user.Password.Equals(password) && user.UserType == usertype) 
+            {
+                return user;
+            }
+            return null;
+        }
+
+        public bool CreateUser(string name, string password, UserType usertype)
+        {
+            try
+            {
+                var user = new User
+                {
+                    Name = name,
+                    Password = password,
+                    UserType = usertype
+                };
+                if (_userRepository.GetUser(name) != null)
+                    return false;
+                _userRepository.CreateUser(user);
+                return true;
+            }
+            catch (Exception ex) 
+            {
+                Console.WriteLine($"Error creating a user: {ex}");
```

Comment:
same comment as above, please apply comments you receive in one place across the whole PR ;)